.readthedocs-custom-redirects.yml

```diff
@@ -0,0 +1,15 @@
+# This file is read by tools/update-rtd-redirects.py.
+# It is related to Read the Docs, but is not a file processed by the platform.
+
+/dev/news-entry-failure: >-
+  https://pip.pypa.io/en/stable/development/contributing/#news-entries
+/errors/resolution-impossible: >-
+  https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
+/surveys/backtracking: >-
+  https://forms.gle/LkZP95S4CfqBAU1N6
+/warnings/backtracking: >-
+  https://pip.pypa.io/en/stable/topics/dependency-resolution/#possible-ways-to-reduce-backtracking
```

Comment:
Is the stable/latest difference intended? I feel the /dev/ one should use latest, while the other two should use stable.

Reply:
Not really. I want us to fix these issues, but I wanted to do it separately from moving the source of truth into the repository source tree; to make sure that we make changes in a separate step from the migration to this.